build.sbt

```diff
@@ -1,13 +1,28 @@
-import Settings._
-import Testing._
+import Settings.*
+import Testing.*
+import pl.project13.scala.sbt.JmhPlugin
 import spray.revolver.RevolverPlugin
 
 lazy val root = project.in(file("."))
-  .settings(rootSettings:_*)
+  .settings(rootSettings *)
```

Comment:
these first few changes are just to get rid of intellij syntax warnings in this file.